frontend/app/models/encryptable-file.js

```diff
@@ -0,0 +1,36 @@
+import Encryptable from "./encryptable";
+import { attr, belongsTo } from "@ember-data/model";
+
+export default class EncryptableFile extends Encryptable {
+  @attr file;
+  @belongsTo("encryptable-credential") encryptableCredential;
+
+  async save() {
+    if (this.isDeleted) {
+      return super.save();
+    }
+    const url = `/api/encryptables`;
+    const credential_id = await this.encryptableCredential.get("id");
```

Comment:
javascript uses camel case, not snake case for variables and methods
```suggestion
    const credentialId = await this.encryptableCredential.get("id");
```